docs/source/features/metrics.md

```diff
@@ -57,6 +57,22 @@ You can set environment variable values on the command line as seen below, or wi
 ENGINE_API_KEY=YOUR_API_KEY ENGINE_SCHEMA_TAG=development node start-server.js
 ```
 
+### Debugging
```

Comment:
Expand to "Debugging Graph Manager reporting

Reply:
Resolved via 10dc7c1d.